### DISCUSSION TITLE
Print'ing messages to console (New Skript Code Debug Method)

### DISCUSSION THREAD

**Discussion by EquipableMC** [SkriptLang/Skript#6648] [Suggestions]
So I made this suggestion on the feedback forums and Sovde said to take it here so I can get more opinions, so I am.

We all use "broadcast" to debug or "send "x" to console" for debugging, but I was thinking we could also have ways like the ways provided below:

```
command /test:
	trigger:
		print in "test"
		send a fatal error "test" to the console
		log a warning "test"
```
![image](https://github.com/SkriptLang/Skript/assets/66171067/0e592892-3e14-42c6-b554-f897476156eb)

As shown in the screenshot, it sends a regular info message for "test" in the next line sends a fatal error with the text "test" to console, and a warning message to console. I feel like this could make debugging easier and make it stand out more if you have a lot of plugin messages in console, and it could make your debug stand out more.

This discussion was mainly to bring up a new way of debugging issues in your own skript code. This idea originated from skript-logs but I thought it would be cool to have in vanilla skript.

Obviously the syntax can be changed to make it easier for skript users, but this is what was in skript-logs.

**Comment by ffogells** [comment]
If this still doesnt exist this would be great.
even if its just a skript plugin able to do this.

**Reply by sovdeeth** [reply]
It has been merged in 2.9
